pkg/api/types.go

```diff
@@ -1020,12 +1020,12 @@ const (
 )
 
 type NodeCondition struct {
-	Type               NodeConditionType `json:"type"`
-	Status             ConditionStatus   `json:"status"`
-	LastProbeTime      util.Time         `json:"lastProbeTime,omitempty"`
-	LastTransitionTime util.Time         `json:"lastTransitionTime,omitempty"`
-	Reason             string            `json:"reason,omitempty"`
-	Message            string            `json:"message,omitempty"`
+	Type                    NodeConditionType `json:"type"`
+	Status                  ConditionStatus   `json:"status"`
+	LastSuccessfulProbeTime util.Time         `json:"lastSuccessfulProbeTime,omitempty"`
```

Comment:
Right - best idea was by @fgrzadkowski, so I'm renaming it to LastHeartbeatTime.